Looms.PoS/Looms.PoS.Application/Features/Reservation/Commands/UpdateReservation/UpdateReservationRequestValidator.cs

```diff
@@ -0,0 +1,29 @@
+using FluentValidation;
+using Looms.PoS.Application.Models.Requests.Reservation;
+using Looms.PoS.Application.Utilities.Validators;
+
+namespace Looms.PoS.Application.Features.Reservation.Commands.UpdateReservation;
+
+public class UpdateReservationRequestValidator : AbstractValidator<UpdateReservationRequest>
+{
+    public UpdateReservationRequestValidator()
+    {               
+        RuleFor(x => x.AppointmentTime)
+            .Cascade(CascadeMode.Stop)
+            .MustBeValidDateTime()
+            .MustBeWithinBusinessHours();
```

Comment:
must be in the future

Reply:
validated that it would be in the future